support/SubmitScripts/Frontera.sh

```diff
@@ -28,26 +28,82 @@
 # To submit the script to the queue run:
 #   sbatch Frontera.sh
 
-# UPDATE THESE!!!
-export SPECTRE_EXECUTABLE=./bin/ParallelInfo
-export SPECTRE_INPUT_FILE=./Test.yaml
+# Replace these paths with the path to your build directory, to the source root
+# directory, the spectre dependencies module directory, and to the directory
+# where you want the output to appear, i.e. the run directory.
+# E.g., if you cloned spectre in your home directory, set
+# SPECTRE_BUILD_DIR to ${HOME}/spectre/build. If you want to run in a
+# directory called "Run" in the current directory, set
+# SPECTRE_RUN_DIR to ${PWD}/Run
+# On Frontera, you'll most likely have cloned spectre to scratch, the $HOME
+# option won't work, so make sure to write out full path from scratch.
+export SPECTRE_BUILD_DIR=${HOME}/Codes/spectre/spectre/build_clang
+export SPECTRE_MODULE_DIR=${HOME}/Codes/spectre_deps/modules/
+export SPECTRE_RUN_DIR=${PWD}/Run
+
+# Choose the executable and input file to run
+# To use an input file in the current directory, set
+# SPECTRE_INPUT_FILE to ${PWD}/InputFileName.yaml
+export SPECTRE_EXECUTABLE=${SPECTRE_BUILD_DIR}/bin/EvolveGhKerrSchild
+export SPECTRE_INPUT_FILE=${PWD}/KerrTest
+
+# These commands load the relevant modules and cd into the run directory,
+# creating it if it doesn't exist
+source ${SPECTRE_BUILD_DIR}/../support/Environments/frontera_gcc.sh
+module use ${SPECTRE_MODULE_DIR}
+spectre_load_modules
+module list
+
+mkdir -p ${SPECTRE_RUN_DIR}
+cd ${SPECTRE_RUN_DIR}
+
+# Copy the input file into the run directory, to preserve it
+cp ${SPECTRE_INPUT_FILE} ${SPECTRE_RUN_DIR}/
 
 # Print out diagnostic info
 module list 2>&1
 pwd
 date
 
-echo ""
-echo ""
-echo ""
-
 export IBRUN_TASKS_PER_NODE=1
 
+checkpoints=0
```

Comment:
instead of defining these, have this script check for a restart argument and a checkpoint number that can be used in an if statement below to submit either the original run or a restart starting from the given checkpoint number